sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -52,7 +56,9 @@ case class ColumnarOverrideRules() extends ColumnarRule with Logging {
  */
 class SQLExecPlugin extends (SparkSessionExtensions => Unit) with Logging {
   override def apply(extensions: SparkSessionExtensions): Unit = {
-    logWarning("Installing extensions to enable rapids GPU SQL support." +
+    val pluginVersion = RapidsPluginUtils.getPluginVersion
+    val cudfVersion = RapidsPluginUtils.getClasspathCudfVersion
+    logWarning(s"RAPIDS Accelerator $pluginVersion using cudf $cudfVersion." +
```

Comment:
I would suggest to log the entire build info, i.e all properties:
```scala
scala> com.nvidia.spark.rapids.RapidsPluginUtils.getPluginProps
res2: java.util.Properties = {version=0.5.0-SNAPSHOT, user=gshegalov, url=git@github.com:NVIDIA/spark-rapids.git, date=2021-04-14T18:55:56Z, revision=122097be00609a525cc756a9fba707685c789710, cudf_version=0.19-SNAPSHOT, branch=log-version}
```
Something to the tune:
```scala
logWarning(s"RAPIDS Accelerator [buildInfo ${RapidsPluginUtils.getPluginProps}] ..." 
```
Also let us update https://github.com/NVIDIA/spark-rapids/blob/branch-0.5/docs/configs.md#rapids-accelerator-for-apache-spark-configuration on how to verify the plugin was loaded using logs and REPL

Reply:
IMO this is _far_ too much detail and clogs up the output with user-confusing details.  Instead of a brief message showing the version, now I have to essentially parse a long line to figure out what versions are really used here.  For example, is the cudf version what `cudf_version` is reporting or the version later on the line after `using cudf`?  It's the latter, but how will the user know that?

I think it might be useful to log the git commit hash, possibly in parentheses, after the version number for both the plugin and cudf, but I'm not a fan of logging all the other stuff.  If we really need that, we can fish the properties file from the jar.

Reply:
I'd rather overlog. We don't need redundant `using cudf ` obviously if we agree to log all the properties.  Git commit is a reasonable compromise.  But given that it's OSS and there can be non-deterministic build related issues,  I'd rather see more than less diagnostics. Deferring customers to fishing in the jars might not be feasible in many environments.

Reply:
If we want to point to OSS, Spark has this properties file but does not dump it on startup.  I don't think we should either.

Reply:
> This appears on the driver startup every time. IMO it's unprofessional and excessive to log raw properties like this.

we can style/relabel them for clarity but I don't find it unprofessional and/or excessive  even as is.

Reply:
> If we want to point to OSS, Spark has this properties file but does not dump it on startup. I don't think we should either.

I meant that everyone can build OSS, so git commit and version is not sufficient to identify jars.  I was not talking about Spark as a reference.

Reply:
I kept the short version logging at the WARN level but now also log the full build props at INFO level on both the driver and executor.

Reply:
Maybe for a followup PR: Instead of logWarning in interactive shells for Scala, Python, R we can `println` this string 
based on 
- main class in sun.java.command system property
- `spark.app.name` https://github.com/apache/spark/blob/60e324aa9f071260f130780fde7438911b0aecee/bin/pyspark#L68
- `spark.repl.class.outputDir` https://github.com/apache/spark/blob/3be552ccc8d26089881229edc034d2ebf2e75511/repl/src/main/scala-2.12/org/apache/spark/repl/Main.scala#L92
...